### PR TITLE
Add test consent manager scope

### PR DESCRIPTION
**Pull request by anotherminh**
## Related Issues

- Part of https://transcend.height.app/T-26244

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: src/scopes.ts [hunk 1]
```diff
@@ -559,11 +561,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
...
+  [ScopeName.DeployTestConsentManager]: {
+    dependencies: [ScopeName.ViewConsentManager],
+    description:
+      // eslint-disable-next-line max-len
+      'Ability to publish changes to the test Consent Manager bundle. This changes the code contents of airgap.js and attempts to invalidate the CDN.',
+    title: 'Deploy Test Consent Manager',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.ConsentManager],
+  },
   [ScopeName.DeployConsentManager]: {
```
Comment (anotherminh) [inline src/scopes.ts:573]: @michaelfarrell76 should this one includes the test scope too or should they be completely separate?

Comment (anotherminh) [reply src/scopes.ts:573]: For now I tried to make it so that prod includes perms to deploy test as well

**Review by MadDataScience — DISMISSED** [review]
It's not clear to me how these different scopes are enforced. Is that coming in a separate PR or am I missing something?

**Comment by anotherminh** [comment]
@MadDataScience yes that would be in the monorepo. All scopes are listed here in privacy-types and then we implement them in the monorepo

**Review by MadDataScience — APPROVED** [review]

**Pull request merged**